src/modules/reports/models/reports.dto.ts

```diff
@@ -0,0 +1,4 @@
+export class DatesDTO {
```

Comment:
Could you change this to a better name? I mean could be GetPatientAppointmentsReportsDTO